### ISSUE TITLE
Branching and Versioning

### ISSUE THREAD

**Issue by mujeerhashmi** [earthians/marley#186]
I think we need to follow the same strategy for branch & version management like HRMS. Kindly refer https://discuss.frappe.io/t/frappe-hr-changes-to-branching-and-versioning/100658/1

**Comment by aakvatech** [comment]
Agreed. We have also implemented mergify and working the way it's done in ERPNext repo. If you need, we can have a session on that.

**Comment by akurungadam** [comment]
the branching and versioning is in place now, we are yet to add mergify and automatic release actions. we have done a bit of ground work looking at frappe and erpnext actions, but thanks @aakvatech i'll connect with you on tg. Or better if you would like to come up with a pr?

**Comment by aakvatech** [comment]
Elius from our team can assist
